api-doc/swagger_v1.yaml

```
@@ -48,6 +48,8 @@ components:
         thumbnail:
           type: string
           nullable: true
+        status:
+          type: string
```

```suggestion
        status:
          type: string
          enum: ['past', 'live', 'upcoming', 'missing', 'new']
```
Add possible values, makes it easier to work with the API
Don't think example would be appropiate here.